modal/_container_io_manager.py

```diff
@@ -1003,7 +1003,7 @@ async def interact(self, from_breakpoint: bool = False):
         try:
             await self._client.stub.FunctionStartPtyShell(Empty())
         except Exception as e:
-            print("Error: Failed to start PTY shell.")
```

Comment:
@thecodingwizard am I missing the reason this is a `print` not a log?

Reply:
Nope -- it should be a log, sorry 😓